examples/all-clusters-app/esp32/main/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+#
+#    Copyright (c) 2021 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#    Description:
+#      Component makefile for the ESP32 demo application.
+#
+# (Uses default behaviour of compiling all source files in directory, adding 'include' to include path.)
```

Comment:
Hi @bzbarsky-apple Yes, I agree with your point. Going ahead, I expect that we would support only CMake (and remove the automake support once all examples/documenation is updated and tested. Although, for now, we will add a comment so that there are no issues. I will raise a separate PR to address this.